ledger-tool/src/main.rs

```diff
@@ -1463,6 +1469,21 @@ fn main() {
                         .value_name("ENDING_SLOT")
                         .help("Ending slot for minimized snapshot creation"),
                 )
+                .arg(
+                    Arg::with_name("no_recalculate_accounts_lt_hash")
```

Comment:
Kind of curious about default behavior here.
Is it better to have not recalculate by default? Seems that was effectively the original intention of minimized snapshots - make sure you get same result as network but just faster to replicate replay of slot range.

recaclulating it serves a different purpose - it's effectively creating a new self-contained snapshot to start from.

wdyt?